### DISCUSSION TITLE
Update of the Czech localization for 2023

### DISCUSSION THREAD

**Discussion by MikkCZ** [pretalx/pretalx#1448] [General]
Hi.

In 2023, the Czech open-source conferences are hopefully coming back to the previous form and glory after COVID hiatus. I started working on update of the localization, which is now ~still below 30%~ around 50 % of completion. What is the expected threshold for the localization completeness before it will be first shipped and enabled in Pretalx, please?

**Comment by rixx** [comment]
Merges are currently intermittent / when things change. I'll merge the Czech localisation in #1451, and support was added in 7f64ea2d and 939f4756.
